internal/cmd/config.go

```diff
@@ -905,4 +909,13 @@ traefik:
 	v.SetDefault("telemetry::debug", true)
 
 	v.SetDefault("integratedservice::v2", false)
+
+	// Integrated Service Operator
+	v.SetDefault("isoperator::enabled", true)
+	v.SetDefault("isoperator::repourl", "")
+	v.SetDefault("isoperator::reponame", "")
+	v.SetDefault("isoperator::chartversion", "latest")
```

Comment:
do we automatically handle the "latest" version?

Reply:
no, this is a draft only...